src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocCommentsTokenTypes.java

```diff
@@ -881,7 +881,36 @@ public final class JavadocCommentsTokenTypes {
     public static final int EXTENDS = JavadocCommentsLexer.EXTENDS;
 
     /**
-     * Keyword {@code super} in type parameters.
+     * {@code SUPER} represents the {@code super} keyword inside a generic
+     * wildcard bound (e.g., {@code ? super Number}).
+     *
+     * <p><b>Example:</b> {@link java.util.List<? super Integer> list
+     * of any supertype of Integer}</p>
```

Comment:
```suggestion
     * <p><b>Example:</b> {@code {@link java.util.List<? super Integer> list}
     * of any supertype of Integer}</p>
```

Can you add the whole link example inside a code tag as we normally do in other examples?

I believe this will make IDEA inspection green also.

Reply:
I tried that as well, that doesn't work either

Reply:
encode `<` as `&lt;`  and `>` as `&gt;`

Reply:
Escaping is not working https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocCommentsTokenTypes.html#EXTENDS

Reply:
@romani @mahfouz72 Any suggestions here?